tensorflow/core/kernels/conv_ops_gpu_3.cu.cc

```diff
@@ -631,6 +690,10 @@ const std::vector<std::pair<int, int>>& GetTileSizesFrontier() {
   return *frontier;
```

Comment:
I think the assert above needs to be a LOG(FATAL), for the reasons in my previous review.